src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -88,19 +97,23 @@ def generate_MCP_1_default_config(self, filename):
         'airInterface': device_8['airInterface'],
         'installationParam': device_8['installationParam'],
         'measCapability': device_8['measCapability']
-    }    
-   
-    conditionals = {'registrationData': [conditionals_device_2, conditionals_device_8]} 
+    }
+
+    conditionals = {'registrationData': [conditionals_device_2, conditionals_device_4, conditionals_device_8]}
```

Comment:
Depending on when PR #479 is submitted, this should probably be changed.

Reply:
thanks, we will wait on that approval and we will help review it.